pandas/plotting/_matplotlib/boxplot.py

```diff
@@ -75,8 +75,8 @@ def _validate_color_args(self):
                     if key not in valid_keys:
                         raise ValueError(
                             "color dict contains invalid "
-                            "key '{0}' "
-                            "The key must be either {1}".format(key, valid_keys)
+                            f"key '{key}' "
+                            f"The key must be either {valid_keys}"
```

Comment:
These two should just be on the same line.  Also add "." after ``'{key}'``.

Reply:
Agreed, though I moved the key line to the previous line, makes it more readable.